pages/common/ngrep.md

```diff
@@ -0,0 +1,15 @@
+# ngrep
+
+> Ngrep is a pcap-aware tool that will allow you to specify extended regular or hexadecimal expressions to match against data payloads of packets.
```

Comment:
This description is too technical and jargon-y. Something closer to the description you used in the PR ("an easy-to-use basic packet sniffing tool.") would match the ethos of the tldr project better.

For example: is PCAP a critical qualifier to describe what this command does? It sounds like a technical feature that could be left to a more detailed description (e.g. in the manpage).